### DISCUSSION TITLE
split a file into several files according to an invoice number

### DISCUSSION THREAD

**Discussion by Larbo53** [jsvine/pdfplumber#580] [Ask for help with specific PDFs]
Hello,

I would like to cut a pdf that contains several invoices in several pages, into as many pdf files with the same formatting, based on the invoice number.
What is the solution, knowing that I have identified the coordinates of the field containing the invoice number?
Thank you.
Sincerely

**Comment by samkit-jain** [comment]
Hi @Larbo53 Appreciate your interest in the library. Pdfplumber does not support writing PDF files. It is focused on extracting information from PDFs, rather than creating/modifying them. You may look at tools like [PyPDF2](https://pypi.org/project/PyPDF2/) for splitting a PDF.

**Reply by Larbo53** [reply]
Good morning,

here is a solution : with python, I create a dictionary with as key the invoice number and as value a list with the pages number of the document. This solution allows me to remove the blank pages as well as the pages containing the general sales conditions.
Thank you.
Sincerely.
